build-repo

```diff
@@ -11,6 +11,15 @@ package=$1
 
 chown -R makepkg:users /build
 
-cd /tmp && sudo -u makepkg asp checkout $package && mv $package/trunk/* /build && cd /
+cd /tmp
+sudo -u makepkg asp checkout "$package"
+mv "$package"/repos/{core,extra,community}-{any,x86_64}/* /build 2>/dev/null || true
+cd /
```

Comment:
I've pushed a potential fix. I think your solution does cover all cases, but I couldn't figure out a way to make mv be happy with its one match (it matches one combination, then prints five could not stat errors and exits with an error status), so I muted and force-trued it. Quite hacky but I think it works.
According to my tests it correctly builds for all three repos and both architectures, and chooses extra over testing.

Reply:
I'm okay with this solution, thanks for testing! 🙂